config/config.yaml

```diff
@@ -175,9 +175,13 @@ policy:
         configfile: portal.conf
         versionpackage: github.com/TykTechnologies/portal/model/version
         baseimage: debian:bookworm-slim
-        buildenv: 1.19-bullseye
+        buildenv: 1.21-bullseye
         branches:
           master:
+            baseimage: debian:bookworm-slim
+            buildenv: 1.21-bullseye
+          release-1.8.0:
```

Comment:
@tbuchaillot this is probably not required. We only reference lts branches here. 
1.8.0 looks like a release our code freeze branches for which the releng code is frozen and should not be updated anymore.